### PR TITLE
Add minimize option to postcss and sass blocks

### PR DESCRIPTION
**Pull request by sapegin**
For consistency with the `css` block.

### CODE REVIEW

File: packages/postcss/README.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ module.exports = {
 
 ## Options
 
+#### minimize *(optional)*
+Enable CSS minification.
```
Comment (vlad-zhukov) [inline packages/postcss/README.md]: I think it should say that it's handled by the `css-loader`.

Comment (vlad-zhukov) [reply packages/postcss/README.md]: Also need to add this options to sass block readme as well.

Comment (sapegin) [reply packages/postcss/README.md]: Good points — done ;-)

File: packages/postcss/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # @webpack-blocks/postcss - Changelog
 
+## 1.0.0-beta.5
```
Comment (andywer) [inline packages/postcss/CHANGELOG.md]: `Next release` might be better. The next release could just as well be `1.0.0-rc`.

**Pull request merged**